.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*.go,Makefile]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+indent_style = space
```

Comment:
tabs are required in Makefiles

Reply:
I wonder if this editorconfig isn't taking effect in the Makefile? It doesn't seem to do anything? (I think I installed the correct extension).

Reply:
yes, it's not expanded. fixed in #26